Automattic/wp-calypso
# Performance Profiler: Improve LLM recommendations area

Related to [Performance Profiler: LLM implementation](https://github.com/Automattic/dotcom-forge/issues/8825)

## Proposed Changes

* Update the feature flag for LLM to true as this is the default behavior now
  * I've turned this feature to true even in environments where the Performance Profiler is disabled as this should already be the default behavior when it gets activated
*  Show a messaging with a rotating icon when we are loading an IA message 
* Show a message stating the content was generated with IA 
* Show a survey in the end
  * Currently the survey is generating a track event

## Why are these changes being made?

To better display LLM messages generated

## Testing Instructions

* Go to `/speed-test-tool?url=:site`. Ex: `/speed-test-tool?url=wordpress.com`
* Go to the `Recommendations` section
* Click on a item and see if the `Loading state` is properly displayed
* After loading check if the answer has the two sections: `"Why is this important?"` and `"How to fix this?"`
* Check if there is a box below saying `Generated with IA`

![CleanShot 2024-09-05 at 16 53 57](https://github.com/user-attachments/assets/aa4888d6-6f11-4dde-8a61-63645cef232a)


## Pre-merge Checklist
